components/scream/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -240,7 +242,7 @@ void P3Microphysics::run_impl (const Real dt)
 
   // Run p3 main
   P3F::p3_main(prog_state, diag_inputs, diag_outputs, infrastructure,
-                                       history_only, m_num_cols, m_num_levs);
+                                       history_only, deprecated, m_num_cols, m_num_levs);
```

Comment:
If I understand correctly, you're still outputting these variables we don't want. Do we really need to do this? I thought Conrad's strategy was to still produce these variables in F90, but not to actually compare them against anything anymore... so the C++ wouldn't need to carry them anymore. Am I missing something?